modules/renter/contractor/contractor.go

```diff
@@ -168,5 +168,8 @@ func newContractor(cs consensusSet, w wallet, tp transactionPool, hdb hostDB, p
 		return nil, errors.New("contractor subscription failed: " + err.Error())
 	}
 
+	// spawn uptime loop
+	go c.threadedMonitorUptime()
```

Comment:
Oh, rather than do it in the update function, since we aren't actually deleting contracts, we might just want to change the logic that the renter uses when it's counting the number of contracts that it currently has.